mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -371,10 +375,10 @@ inline Value emitLoadLogic(OpBuilder &b, Location loc, MemRefType sourceType,
       Value upperBoundCheckOp =
           b.create<CmpIOp>(loc, CmpIPredicate::slt, coord, upperBoundOp);
       Value withinBoundInOneDimOp =
-          b.create<AndOp>(loc, lowerBoundCheckOp, upperBoundCheckOp);
+        b.create<LLVM::AndOp>(loc, lowerBoundCheckOp, upperBoundCheckOp);
```

Comment:
I'm not sure that's the right fix. Wouldn't we want something in `arith` instead?

Reply:
Oh.  I removed some "arith::" prefixes, which shortened lines enough that the next line could fit, and I didn't merge them.  Foo.  clang-format lacks aesthetics.